### ISSUE TITLE
It's possible a promise in unzipCUB3ds.js gets rejected without logging

### ISSUE THREAD

**Issue by strycore** [anthill/city-core#90]
The README estimates the extraction time to 5 minutes on "normal hardware"but I've launched the extraction procedure more than an hour ago and it's still working at it!

My guess is that the operation takes around 5 minutes *per exported tile* (I have exported something like 25 tiles), the docs should probably mention that.

**Comment by DavidBruant** [comment]
True. One my machine, it's about one minute per exported tile.
What hardware/OS do you have? 

(I'll change the doc right away)

**Comment by strycore** [comment]
I'm running this on a Core 2 Quad @ 2.66Ghz with Ubuntu 14.04 x64.
The operation is still running btw and I don't see any generated files, either in /tmp/tmp-15284e5gphsg or in front/data (both folder empty, except for a .gitkeep in data)

**Comment by DavidBruant** [comment]
Something went wrong. It's possible a promise got rejected and we don't log its error.
I'll give it another try.

**Comment by DavidBruant** [comment]
We're finishing a major refactoring, so it's possible things don't all go well, sorry about that.
Repurposing the issue.

**Comment by strycore** [comment]
aaah, much better :D The extraction is quite fast (2~10 seconds/tile)

**Comment by DavidBruant** [comment]
what is much better?
Does it work because you did git pull?

**Comment by strycore** [comment]
No, I just stopped the process (which had been eating 100% of a core doing nothing) and relaunched it without changing anything. The extraction process worked as expected.

**Comment by DavidBruant** [comment]
Ok. Closing then...

**Comment by strycore** [comment]
Probably not worth opening a new issue: there's a missing 'run' between npm and 'start-dev'. 
Got the project to run but it's completely unusable on this machine, will try again at home with a proper GPU.

**Comment by DavidBruant** [comment]
> Probably not worth opening a new issue: there's a missing 'run' between npm and 'start-dev'. 

Fixed. Thanks :-)

**Comment by vallettea** [comment]
I jsut stumbled upon the same issue node unzipCUB...

**Comment by vallettea** [comment]
Related from @iOiurson (french):

Grosso modo, le problème n’apparaît que lors de l’unzip de gros fichiers .zip.
Je sais pas exactement le seuil, mais jusqu’à 110Mo zippé, ca marche, >250Mo, ca ne marche pas.

En fait, le pb est dans unzipInTmpDirectory. Je me rappelle on avait discuté de ça fin décembre parce que j’arrivais pas à unzip une tile avec textures.
Je pense que le problème est le même: le fichier est trop gros.

Du coup, l’évènement « finish » se lance, mais pas l’évènement « close », ou se trouve le def.resolve.
Quand le fichier est petit, finish trigger, puis close, donc on a pas de problème.
Quand le fichier est plus gros, donc, close ne trigger jamais.

Pour fixer les idées, sur bordeaux centre, il y a une cinquante de fichiers zip à extraire, mais seulement 40 se retrouvent dans le tmp.

**Comment by DavidBruant** [comment]
Maybe duplicate of #76

**Comment by vallettea** [comment]
indeed but let's close this one and see it fixes the other one
